### ISSUE TITLE
Virus scanner false positive

### ISSUE THREAD

**Issue by infinitel8p** [f1multiviewer/issue-tracker#277]
Kaspersky and VirusTotal flags the .exe as Trojan.

[VirusTotal Scan for 'MultiViewer for F1.exe'](https://www.virustotal.com/gui/file/c09475ef80fea6d480001a3a11fd3f63180a70482613b6320f470cd48a5c296a/detection)

![image](https://user-images.githubusercontent.com/50703696/199075069-d04a9720-6ca6-453f-9b36-6151ab5aed6f.png)

**Comment by f1multiviewer** [comment]
Hey @infinitel8p, unfortunately, this is a known issue, likely related to the fact that this app uses Electron, and isn't 'codesigned' on Windows, which makes it similar to other Electron-based apps, some of which may be malware, but this is not. I'm still looking at this, but so far, updating Electron didn't seem to fix this, and code-signing certificates require me to give up my anonymity, something I'm not willing to do at this time.

Potentially, a member of the community might step in and provide a code-signing certificate for MultiViewer to use, similar to how another member did this for macOS, but so far, nobody offered this.

**Comment by f1multiviewer** [comment]
![image](https://user-images.githubusercontent.com/106821994/199229058-79403c4d-8335-438e-b5a1-5498bf341c46.png)

Reached out to Gridinsoft and they say it should be corrected soon 🙏

**Comment by f1multiviewer** [comment]
VirusTotal replied this:

> Hello,
>  
> VirusTotal only aggregates data from a variety of vendors. We produce no verdicts of our own and as such, we can’t modify these results. We are not intended to be an authoritative reputation engine, but rather provide intelligence and context to users so that they can make the best decision. 1/60 and even 5/60 doesn’t automatically mean “Bad”, and 0/60 doesn't always mean good. Each decision on whether something is malicious ultimately the responsibility of users or the security vendors who use the data to improve their services.
>  
> Below are the steps to take if you want to fix a false positive report.
>  
> 1. If the false positive is for a File or a URL try re-scanning first.
> 2. If the false positive persists please reach out to the vendor that is producing it providing a link to the Virustotal report. List of vendor contacts can be found at the table below.
> 3. If you do reach out to us, we will tell you to reach out to the vendors, as we will not be able to modify any results of scans.
> 4. We cache some URL results. If vendors have confirmed removal but a URL is still detecting, please reach out to us along with evidence that the vendor has removed your URL from the blacklist, as we may be able to speed up propagation.

...but this isn't particularly helpful. I'll wait for GridinSoft to respond, hopefully something went wrong while removing the false-positive, and it's an easy fix 🙏

**Comment by f1multiviewer** [comment]
Reached out to them again, they say they'll fix it ASAP, but it's still being flagged. I'm not sure this will be resolved any time soon, and there's not much I can do from my end to prevent it. I'll check up with them in a week if it's not resolved by then.

![support-antimalware.com](https://user-images.githubusercontent.com/106821994/200552994-28b5da2b-4451-47b9-bb07-e73a77d0c186.png)

**Comment by f1multiviewer** [comment]
it does seem that at least kaspersky no longer flags it:

https://opentip.kaspersky.com/C09475EF80FEA6D480001A3A11FD3F63180A70482613B6320F470CD48A5C296A/results

**Comment by infinitel8p** [comment]
thanks for the follow-up, ill check it later as well and give you feedback 👍

**Comment by infinitel8p** [comment]
Issue seems to be solved, made a full scan with kaspersky after installing multiviewer and got no errors

**Comment by f1multiviewer** [comment]
Not just that! It also is clear now on GridinSoft/VirusTotal: https://www.virustotal.com/gui/file/7d1427bbbd5b26e5504c04242d560aa4fe11f85fa83d54f7e4f3008fd6ef9b97

That means it's completely fixed! 💚

**Comment by f1multiviewer** [comment]
Unfortunately, I'm reopening this as it remains an issue, and an update from GridinSoft confirming that without a code-signing certificate, I won't be able to be unlisted as a false-positive.

<img width="893" alt="image" src="https://user-images.githubusercontent.com/106821994/201651099-865bb5b3-c50d-43a5-a66d-040e0e65e80b.png">

**Comment by LapsTimeOFF** [comment]
Any news ?

**Comment by f1multiviewer** [comment]
Yes, I've gotten an offer from a member to buy us a code-signing certificate, which will be introduced just before the season starts. I thought doing it during the off-season was a bit wasteful ;)

**Comment by LapsTimeOFF** [comment]
Perfect ! Let's hope it will correct the issue !

**Comment by LapsTimeOFF** [comment]
And for the problem on the ticket I posted a few weeks ago by Avast blocking the domain, any news on that ?https://discord.com/channels/982375740764983366/1050075400946057336

**Comment by nick-potts** [comment]
This should be able to get closed now ;)

**Comment by wizard4u** [comment]
Avast flags it still as virus

**Comment by f1multiviewer** [comment]
@wizard4u yeah, we're still working on getting code-signing to work, but it's rather difficult, as it seems to break Castlabs' EVS signing, which is required for widevine playback.
